src/Snackbar/Snackbar.js

```diff
@@ -169,7 +170,7 @@ type Props = DefaultProps & {
    * @param {event} event The event that triggered the close request
    * @param {string} reason Can be:`"timeout"` (`autoHideDuration` expired) or: `"clickaway"`
    */
-  onRequestClose?: (event: ?SyntheticUIEvent, reason: string) => void,
+  onRequestClose?: (event: ?Event, reason: string) => void,
```

Comment:
If not a `SyntheticUIEvent`, then `SyntheticEvent`, these are the types used by react.  `Event` is a dom type, and technically react `Event !== SyntheticEvent`